### PR TITLE
Add techpreview release notes

### PR DESCRIPTION
**Pull request by dprince**

### CODE REVIEW

File: docs/RELEASE-techpreview.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Overview
+--------
+The OSP Director Operator is an OpenShift Container Platform (OCP) “Operator” that streamlines the installation of OpenStack on OpenShift. The OSP Director Operator creates a set of Custom Resource Definitions (CRDs) on top of OpenShift to manage resources normally created by the TripleO Undercloud. These CRDs are split into two types for hardware provisioning and software configuration which aligns well with TripleO's Undercloud Lite architecture.
+
+These CRDs allow administrators to provision and scale out virtual and baremetal hardware for an OSP deployment. Virtual machines for the OSP control plane are provisioned via Kubevirt(OpenShift Virtualization). Baremetal for OSP compute hosts is provisioned via Metal3. And OvercloudNet provides a simple IPAM solution for IPv4 and IPv6 static IP assignments.
+
+Supported Features
+------------------
+
+The following table highlights features supported today in OSP 16.2 compared with future planned feature work for OSP 17 (GA).
```
Comment (abays) [inline docs/RELEASE-techpreview.md:13]: Should we mention something about HA in the features?

Comment (dprince) [reply docs/RELEASE-techpreview.md:13]: You mean like HA would require a fencing agent?

Comment (abays) [reply docs/RELEASE-techpreview.md:13]: Sure.  Just something to indicate that "full HA" has not yet been addressed.

Comment (dprince) [reply docs/RELEASE-techpreview.md:13]: just pushed an update to include this. thanks!

File: docs/RELEASE-techpreview.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+| Playbook Generator (scale)         | Yes                     | Yes         |
+| Playbook Runner workflow           | No                      | Yes         |
+| DCN/Cells/Multiple Stacks          | No                      | Yes         |
+| IPv4                               | Yes                     | Yes         |
+| IPv6                               | No (untested)           | Yes         |
+| TLS-Everywhere                     | No                      | Yes         |
+| Scale Testing                      | No                      | Yes         |
+| NFV features                       | No                      | Yes         |
+| Overcloud HA (fencing agent)       | No                      | Yes         |
+| ---------------------------------- | ----------------------- | ----------- |
```
Comment (stuggi) [inline docs/RELEASE-techpreview.md:30]: I don't think we need the closing line

https://github.com/dprince/osp-director-operator/blob/techpreview/docs/RELEASE-techpreview.md

Comment (dprince) [reply docs/RELEASE-techpreview.md:30]: sure, I can drop it.

**Review by stuggi — APPROVED** [review]
/lgtm

**Pull request merged**